resources.md

```diff
@@ -4,5 +4,5 @@ title: Resources | Honey Bee Design System
 ---
 ## Resources
 
-- [Honey Bee on Github](https://github.com/UW-Madison-DoIT/honeybee-design-system)
+- [Honey Bee on Github](https://github.com/UW-Madison-DoIT/it-design-system)
```

Comment:
Probably should bump the label of the URL too.

Reply:
Fixed in the latest commit